cloud/migrate-to-cloud/hypershift.md

```diff
@@ -160,8 +164,8 @@ Use this format:
 
     ```bash
     docker run -v$(pwd)/hypershift.yml -ti timescale/hypershift:0.2 clone \
```

Comment:
@Loquacity there should be a space between `-v` and $. 
Reference Evan's command:
```
docker run -v $(pwd)/hypershift.yml:/hypershift.yml -ti timescale/hypershift:0.2 clone \
-s "host=<> user=tsdbadmin port=33870 password=<>" \
-t "host=<> user=tsdbadmin port=33039 password=<>" \
--hypertable /hypershift.yml
```

Reply:
oooh! Good spot! fixed 😄